apps/music/test/unit/player_view_test.js

```diff
@@ -0,0 +1,105 @@
+/* global PlayerView */
+
+'use strict';
+
+require('/js/ui/views/player_view.js');
+require('/shared/js/l10n.js');
+
+suite('Star Rating Bar Accessibility Test', function () {
```

Comment:
Call it something more general: Player View Test (other future tests can go here)